### ISSUE TITLE
Pod - Container Log Switch not Possible due to Javascript Error.

### ISSUE THREAD

**Issue by mixxor** [argoproj/argo-cd#14347]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

I want to look at the logs of my pod - Containers but can’t change the container.

**To Reproduce**

* select a Pod 
* click on “Logs”
* try to change the container from one Container to another Container
* log output does not change
* you can see a Javascript error in the console: 

`Uncaught TypeError: Cannot read properties of undefined (reading 'containers')`

**Expected behavior**

Switch Logs to desired Pod Container.

**Screenshots**

**Version**

```shell
v2.7.1+5e54351
```

**Logs**

```
main.a726808c41da12ea7cc9.js:2 Uncaught TypeError: Cannot read properties of undefined (reading 'containers')
    at onChange (main.a726808c41da12ea7cc9.js:2:1878655)
    at Object.a (main.a726808c41da12ea7cc9.js:2:750751)
    at p (main.a726808c41da12ea7cc9.js:2:750894)
    at main.a726808c41da12ea7cc9.js:2:751040
    at _ (main.a726808c41da12ea7cc9.js:2:751126)
    at rt (main.a726808c41da12ea7cc9.js:2:766594)
    at it (main.a726808c41da12ea7cc9.js:2:766404)
    at st (main.a726808c41da12ea7cc9.js:2:766760)
    at ht (main.a726808c41da12ea7cc9.js:2:767967)
    at F (main.a726808c41da12ea7cc9.js:2:864554)
```

**Comment by crenshaw-dev** [comment]
I'm unable to reproduce the issue in 2.7.6. Is there anything unusual about the pod?

**Comment by mixxor** [comment]
Thank you for taking care of this.

We already solved that issue by Updating to 2.7.2.

Closing the Ticket now.
